runtime/doc/autocmd.txt

```diff
@@ -871,6 +872,8 @@ TermChanged			After the value of 'term' has changed.  Useful
 				for re-loading the syntax file to update the
 				colors, fonts and other terminal-dependent
 				settings.  Executed for all loaded buffers.
+						 {Nvim} *TermClose*
+TermClose			When a terminal buffer ends.
 						 {Nvim} *TermOpen*
 TermOpen			When a terminal buffer is starting.  This can
 				be used to configure the terminal emulator by
```

Comment:
Not sure if it was discussed before, but `TermOpen` and `TermClose` should probably both be mentioned in `vim_diff.txt` (section 4).